contents/docs/libraries/nuxt-js.md

```diff
@@ -106,7 +106,7 @@ npm install --save posthog-js
 
 After that we want to create a app in `plugins/posthog/index.js`
 
-```javascript
+```browser-js
```

Comment:
This is again a case where it isn't really browser-js but some weird third thing, and we don't really want to do a `Vue` snippet type 🤔

Reply:
will change back to js
```suggestion
```javascript
```